apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/authentication/AuthTokenType.java

```diff
@@ -0,0 +1,15 @@
+package edu.pw.apsienrollment.authentication;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.time.Duration;
+
+@RequiredArgsConstructor
+@Getter
+public enum AuthTokenType {
+    ACCESS_TOKEN(Duration.ofHours(2)),
```

Comment:
Duration of access token could be lower (e.g. 10-15 min), but in our system, where we do not use tokens to store access rights (roles, permisions etc.) that can change at any time, 2 hours should be fine.

Reply:
We can adjust it and add removing tokens (by a scheduled job, maybe?) later in development. If everything else if fine, let's merge the PR so we can start working on other user stories